### PR TITLE
integration test for balance

### PR DESCRIPTION
**Pull request by singular-value**
Related to #57

### CODE REVIEW

**Review by vtomole — APPROVED** [review]
LGTM

**Comment by singular-value** [comment]
will fix

**Comment by singular-value** [comment]
(will merge after #57 is merged in)

**Comment by singular-value** [comment]
Seeking review again now that #57 is merged in.

File: qiskit_superstaq/daily_integration_test.py [hunk 1]
```diff
@@ -38,6 +38,10 @@ def test_aqt_compile(provider: qiskit_superstaq.superstaq_provider.SuperstaQProv
     assert provider.aqt_compile([circuit, circuit]).circuits == [expected, expected]
 
 
+def test_get_balance(provider: qiskit_superstaq.superstaq_provider.SuperstaQProvider) -> None:
+    assert isinstance(provider.get_balance(pretty_output=False), float)
```
Comment (richrines1) [inline qiskit_superstaq/daily_integration_test.py:42]: is it worth testing `pretty_output=True` as well?

Comment (singular-value) [reply qiskit_superstaq/daily_integration_test.py:46]: 👍 
Also https://github.com/SupertechLabs/cirq-superstaq/pull/156 for parity

**Review by richrines1 — APPROVED** [review]
lgtm

**Comment by richrines1** [comment]
if we want to be super careful we could also do something like
```
assert re.fullmatch(r"\$\d+\.\d\d", balance_str)
```
but this is probably overkill

**Comment by singular-value** [comment]
> if we want to be super careful we could also do something like
> 
> ```
> assert re.fullmatch(r"\$\d+\.\d\d", balance_str)
> ```
> 
> but this is probably overkill

Probably simple enough that the basic startswith "$" check would suffice IMO, but regex lesson accepted gladly 🐬

**Pull request merged**